srcCpp/perftest_cpp.h

```diff
@@ -141,6 +146,9 @@ class perftest_cpp
     static LARGE_INTEGER _ClockFrequency;
   #endif
 
+    // Priorities for the threads used by perftest and domain participant
+    static PerftestThreadPriorities threadPriorities;
```

Comment:
Do we need this to be static? (I am just worried that if we ever support this for VxWorks it might have the same issue we saw. If it implies much work, lets leave it like this, else, lets make it a parameter of the object, not of the class.

Reply:
Also, thread_priorities